managegroup.php

```diff
@@ -341,7 +341,13 @@
 	{
 		error($lang->group_no_members);
 	}*/
+	
 	$perpage = $mybb->settings['membersperpage'];
```

Comment:
Oh, my bad, I thought it was memberlist.php, which throws an SQL error on the profile fields query with negative `$mybb->settings['membersperpage']`. No reason not to use the `< 1` check nonetheless.